docs/tutorials/simple-nodejs-app-ci-workflow.md

```diff
@@ -0,0 +1,106 @@
+---
+weight: 14
+title: Kev CI workflow example with a simple Node.js App
+---
+
+# CI workflow example with a simple Node.js app  
+
+This example walks through using Kev in a commit based CI pipeline.
+
+You will setup CircleCI to reconcile, render, build, push and deploy to a remote staging environment. 
```

Comment:
am not sure that we should even mention reconcile as it's irrelevant at this stage.